### PR TITLE
2.3 ->  develop

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #

# 👩🏻‍💻 What does this PR do?
Merge 2.3 changes into develop

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] _(e.g.)_ Central Sync server with 1 Legacy Desktop remote site and 1 OMS remote site running this PR
- [ ] _(e.g.)_ This sample datafile: _google drive link_
- [ ] _(e.g.)_ Open a requisition with some lines
- [ ] _(e.g.)_ Make a couple invoices supplying some amount of those lines
- [ ] _(e.g.)_ Review that "issued" column is the sum of the amount already issued in invoices for this requisition

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

**Review by lache-melvin — CHANGES_REQUESTED** [review]
stocktake backend aint compiling?

**Comment by lache-melvin** [comment]
CI shows failing tests now 👀

**Comment by lache-melvin** [comment]
Not sure if they are a result of the merge but should fix 🙏

**Comment by roxy-dao** [comment]
> Not sure if they are a result of the merge but should fix 🙏

Fixed frontend tests... think they've been failing for a while 👀 
the backend one is https://github.com/msupply-foundation/open-msupply/issues/4608 <- 👀

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "2.3.00",
+  "version": "2.3.00-RC5",
```
Comment (lache-melvin) [inline package.json:4]: Should we leave this as 2.3.00? Or is develop now considered 2.4 👀

**Review by lache-melvin — CHANGES_REQUESTED** [review]
Ohh the client tests issue is because they're running against develop rather than your branch! Can we fix here? Should just remove these two lines 🙏 

Screenshot 2024-10-08 at 10 09 01 AM

Otherwise looking great! Thanks Roxy 😁

File: package.json (hunk 1 above)
Comment (roxy-dao) [reply package.json:4]: Can change now or leave. Usually the first person who does a migration changes but it's all goodd

**Review by lache-melvin — APPROVED** [review]
Thank yoouuuu 🚀

**Pull request merged**